pkgs/development/python-modules/cstruct/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "cstruct";
+  version = "5.2";
+
+  src = fetchFromGitHub {
+    owner = "andreax79";
+    repo = "python-cstruct";
+    rev = "v${version}";
+    sha256 = "sha256-Dwogf0mmxFyBV7tPsuKV6gMZLPSCm7YhzqgJNHpaPFA=";
+  };
+
+  nativeBuildInputs = [
```

I guess this should be:

```suggestion
  nativeCheckInputs = [
```

---

Thanks, wasn't aware of nativeCheckInputs yet. :)